plugins/generator-1.20.1/forge-1.20.1/templates/potioneffect.java.ftl

```diff
@@ -31,7 +31,23 @@
 <#-- @formatter:off -->
 <#include "mcitems.ftl">
 <#include "procedures.java.ftl">
-
+<#assign isDefaultCuresPresent = false>

```

Comment:
This is a very weird approach. I don't understand why all these are needed

Reply:
because "DEFAULT_CURES" is equivalent to "MILK" and "PROTECTED_BY_TOTEM" and therefore I made sure that these values ​​were not put together or that it would not be generated by adding the default values ​​I mentioned before